File: AbetOrder/OpenDataAccess/OpenDataAccess.Entity/TablePropertyAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace OpenDataAccess.Entity
{
    public class TablePropertyAttribute : Attribute
    {
        public TablePropertyAttribute()
            : base()
        {
        }

        public string Name { get; set; }

        public string PrimaryKey { get; set; }

        /// <summary>
        /// 不许查询字段名集合
        /// </summary>
        public string NoSelectNames { get; set; }
    }
}

File: JavaWebFramework/src/OpenDataAccess/Entity/TablePropertyAttribute.java
package OpenDataAccess.Entity;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface TablePropertyAttribute {
    public String Name() default "";

    public String WithSql() default "";

    public String PrimaryKey() default "";

    /// 不许查询字段名集合
    public String NoSelectNames() default "";
}
